src/util/ses.ts

```diff
@@ -402,33 +402,33 @@ export async function sendNFTBookPendingClaimEmail({
             title1: titleZh,
             content1: `<p>親愛的${receiverDisplayName || '讀者'}：</p>
             <p>感謝購買<a href="${nftPageURLZh}">《${bookName}》</a>。</p>
-            <p>請<a href="${claimPageURLZh}">登入 Liker Land 網站的書架</a>閱讀你的電子書。
+            <p>請<a href="${claimPageURLZh}">登入 Liker Land 書店的書架</a>閱讀你的電子書。
```

Comment:
"書店的書架" means listing page for me

How about
```suggestion
            <p>請在 Liker Land 書店，到「<a href="${claimPageURLZh}">我的書架</a>」閱讀你的電子書。
```